demo/c/cobra_demo_mic.c

```diff
@@ -84,6 +85,8 @@ static void print_dl_error(const char *message) {
 }
 
 static volatile bool is_interrupted = false;
+static const float alpha  = 0.25f;
```

Comment:
decrease `alpha` as right now it seems that there is a noticeable delay for recognizing voice activities. I've tried 0.15 and it was good to my eye but feel free to experiment more